demonstrations/tutorial_fdhs.metadata.json

```
@@ -0,0 +1,64 @@
+{
+    "title": "Fixed Depth Hamiltonian Simulation via Cartan Decomposition",
+    "authors": [
+        {
+            "username": "Qottmann"
+        }
+    ],
+    "dateOfPublication": "2024-12-13T00:00:00+00:00",
+    "dateOfLastModification": "2024-12-13T00:00:00+00:00",
+    "categories": [
+        "Quantum Computing"
```

Should Algorithms be added as a category?
```suggestion
    "categories": [
        "Quantum Computing",
        "Algorithms"
```

---

Yes good idea 👍  will add ✅